src/Instante/Bootstrap3Renderer/BootstrapRenderer.php

```diff
@@ -5,11 +5,11 @@
 use Nette;
 use Nette\Application\UI\ITemplate;
 use Nette\Forms\Controls;
-use Nette\Bridges\FormsLatte\FormMacros;
+use Nette\Bridges\FormsLatte\Runtime;
 use Nette\Utils\Html;
 
-if (!class_exists('Nette\Bridges\FormsLatte\FormMacros')) {
```

Comment:
I guess aliasing the class is not a good idea - Runtime is not a new name for FormMacros class, it is actually different class that just took over renderFormBegin|End methods. I'd suggest to avoid aliasing and rather try late static binding, like call $formMacros::renderFormBegin|End and filling $formMacros with proper class name.